spec/features/ncr_work_orders_spec.rb

```diff
@@ -450,8 +440,8 @@
         click_on 'Update'
         proposal = Proposal.last
 
-        expect(proposal.approvers.first.email_address).to eq (approver.email_address)
-        expect(proposal.individual_approvals.first.actionable?).to eq (true)
+        expect(proposal.approvers.first.email_address).to eq approver.email_address
+        expect(proposal.individual_approvals.first.actionable?).to eq true
```

Comment:
thoughts on using a predicate matcher? https://www.relishapp.com/rspec/rspec-expectations/v/2-2/docs/matchers/predicate-matchers

`expect(proposal.individual_approvals.first).to be_actionable`

reads nicely!